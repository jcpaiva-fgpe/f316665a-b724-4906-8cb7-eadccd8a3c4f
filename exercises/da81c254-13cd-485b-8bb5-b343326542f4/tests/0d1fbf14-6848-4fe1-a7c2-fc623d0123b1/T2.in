10 0
15 1
