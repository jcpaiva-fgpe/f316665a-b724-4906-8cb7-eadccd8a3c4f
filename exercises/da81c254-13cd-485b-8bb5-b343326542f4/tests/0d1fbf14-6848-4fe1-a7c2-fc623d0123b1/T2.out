Passaram apenas 301 minutos!
Queres dizer, 5 horas e 1 minuto?!
